02-peer-protocol.md

```diff
@@ -914,21 +978,20 @@ This message initiates the v2 channel establishment workflow.
    * [`point`:`delayed_payment_basepoint`]
    * [`point`:`htlc_basepoint`]
    * [`point`:`first_per_commitment_point`]
-   * [`byte`:`channel_flags`]
+   * [`bigsize`:`channel_flags_len`]
+   * [`channel_flags_len*byte`:`channel_flags`]
    * [`opening_tlvs`:`tlvs`]
 
-
 1. `tlv_stream`: `opening_tlvs`
 2. types:
-   1. type: 1 (`option_upfront_shutdown_script`)
+   1. type: 0 (`upfront_shutdown_script`)
```

Comment:
taking out that `option` eh? haha

Reply:
if this is non-optional, should we promote it out of the TLV? I can't remember the story behind this field.

Reply:
I didn't mean that change to make it look like it's not optional, I just wanted to make it consistent with the tlv field in `open_channel`...but the feature in bolt 9 is called `option_upfront_shutdown_script` so I should leave this one unchanged and instead I should change the name in `open_channel` and `accept_channel`!

Reply:
Yep, sounds good to me. Thanks for the explanation.